<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSort">
    <property name="title">
     <string>分拣</string>
    </property>
    <addaction name="actionSort"/>
   </widget>
   <widget class="QMenu" name="menuArrived">
    <property name="title">
     <string>到件</string>
    </property>
    <addaction name="actionContainerNo"/>
   </widget>
   <widget class="QMenu" name="menuSend">
    <property name="title">
     <string>派件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPack">
    <property name="title">
     <string>集包</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>配置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuQueryStatics">
    <property name="title">
     <string>查询统计</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="actionDrop"/>
   </widget>
   <widget class="QMenu" name="menuAlarm">
    <property name="title">
     <string>报警</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>日志</string>
    </property>
    <addaction name="actionRunLog"/>
    <addaction name="actionDetailLog"/>
   </widget>
   <addaction name="menuSort"/>
   <addaction name="menuArrived"/>
   <addaction name="menuSend"/>
   <addaction name="menuPack"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuQueryStatics"/>
   <addaction name="menuTest"/>
   <addaction name="menuAlarm"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSort">
   <property name="text">
    <string>测试落格</string>
   </property>
  </action>
  <action name="actionDrop">
   <property name="text">
    <string>落格</string>
   </property>
  </action>
  <action name="actionContainerNo">
   <property name="text">
    <string>圆通车签号</string>
   </property>
  </action>
  <action name="actionRunLog">
   <property name="text">
    <string>掉落日志</string>
   </property>
  </action>
  <action name="actionDetailLog">
   <property name="text">
    <string>详细日志</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
